<launch>

  <!-- kinova -->
  <arg name="use_urdf" default="true"/>
  <arg name="kinova_robotType" default="j2s6s300" />
  <arg name="kinova_robotName" default="$(arg kinova_robotType)"/>
	<arg name="kinova_robotSerial" default="not_set" />
	<arg name="use_jaco_v1_fingers" default="false" />
	<arg name="feedback_publish_rate" default="0.1" />

  <node name="sim_arm_grasp_robotiq" pkg="sim_grasp" type="sim_arm_grasp_robotiq" output="screen" cwd="node" /> -->
    
  <!-- Realsense -->
  <include file="$(find realsense2_camera)/launch/rs_aligned_depth.launch"/>

  <!-- grasp detection -->
  <node pkg="sim_grasp" type="grasp_detect.py" name="grasp_detect" output="screen" />

  <!-- robotiq -->
  <!-- <node pkg="robotiq_2f_gripper_control" type="Robotiq2FGripperRtuNode.py" name="Robotiq2FGripperRtuNode" output="screen" />
  <node pkg="robotiq_2f_gripper_control" type="Robotiq2FGripperSimpleController_my.py" name="Robotiq2FGripperSimpleController_my" output="screen" /> -->


</launch>
